src/main/java/io/apicurio/datamodels/cmd/commands/DeleteAllExamplesCommand.java

```diff
@@ -73,7 +73,8 @@ public void execute(Document document) {
             return;
         }
 
-        this._oldExamples = new HashMap<>();
+        // Preserve insertion order with LinkedHashMap
+        this._oldExamples = new LinkedHashMap<>();
```

Comment:
This change will preserve ordering in Java... but I wonder what will happen when transpiled.

Reply:
I looked at the TS and it looked like it was transpiled to an array object structure, which should preserve ordering. And I (think) I verified that the TS tests failed without these changes (it was a while ago that I looked though).